packages/cloud_functions/.gitignore

```diff
@@ -0,0 +1,8 @@
+.DS_Store
```

Comment:
I think you can delete all the `.gitignore` files. I think we are trying to keep only one from now on from https://github.com/flutter/plugins/pull/588

Reply:
Done